backend/engine/plugins/github_repo_health/cli/src/rules/helpers.py

```diff
@@ -25,16 +37,22 @@ def evaluate_array_config(config, eval_fn):
 
 
 def add_metadata(passing, check, config={}, error_message=None):
+    id = config.get("id") or check.identifier
     name = config.get("name") or check.name
     description = config.get("description") or check.description
+    severity = config.get("severity")
 
     result = {
-        "type": check.identifier,
+        "id": id,
         "name": name,
         "description": description,
         "pass": passing,
     }
 
+    if severity:
+        # Add severity as an optional configuration
+        result["severity"] = severity
```

Comment:
If the severity is an empty string (a valid severity level) this will evaluate to false and not add the severity to the result. `if severity is not None` might be better.

Reply:
Good catch. I don't think it will matter in the context of Artemis, since no severity will be made into empty string there anyway, but it'd make sense for `"severity": ""` in the config to explicitly set the severity.